sympy/solvers/tests/test_ode.py

```diff
@@ -1491,7 +1491,7 @@ def test_1st_homogeneous_coeff_ode():
 
     sol1 = Eq(log(x), C1 - log(f(x)*sin(f(x)/x)/x))
     sol2 = Eq(log(x), log(C1) + log(cos(f(x)/x) - 1)/2 - log(cos(f(x)/x) + 1)/2)
-    sol3 = Eq(f(x), -exp(C1)*LambertW(-x*exp(1 - C1)))
+    sol3 = Eq(f(x), -exp(C1)*LambertW(-x*exp(-C1 + 1)))
```

Comment:
Why is this change needed?

I wouldn't expect it to make any difference:
```julia
In [29]: C1 = Symbol('C1')                                                                                                        

In [30]: Eq(f(x), -exp(C1)*LambertW(-x*exp(1 - C1)))                                                                              
Out[30]: 
         C₁         ⎛    1 - C₁⎞
f(x) = -ℯ  ⋅LambertW⎝-x⋅ℯ      ⎠

In [31]: Eq(f(x), -exp(C1)*LambertW(-x*exp(-C1 + 1)))                                                                             
Out[31]: 
         C₁         ⎛    1 - C₁⎞
f(x) = -ℯ  ⋅LambertW⎝-x⋅ℯ      ⎠


```

Reply:
Yes both are the same. Actually in previous commits I changed these tests and now It doesn't need to change tests in `test_ode`. You can see in changes from all commits that nothing is being changed in `test_ode`.